include/psa/crypto_config.h

```diff
@@ -763,7 +763,7 @@
  *
  * Module:  library/nist_kw.c
  *
- * Requires: MBEDTLS_AES_C and MBEDTLS_CIPHER_C
+ * Requires: PSA_WANT_ALG_ECB_NO_PADDING and WANT_ALG_AES
```

Comment:
We should update "Requires" to "Auto-enables".

In the past, we used "Requires" because it was up to users to enable dependencies for things they wanted. Now we are doing it for them in the various `crypto_adjust_config` files, so they're no longer required to do it. We can still keep the info here if they're curious, but should not give the impression that there's something they need to do about it.